File: DroidDailyProject/app/src/main/java/six/ca/droiddailyproject/fragment/FragmentOne.java
package six.ca.droiddailyproject.fragment;

import android.content.Intent;
import android.os.Bundle;
import androidx.annotation.Nullable;
import androidx.fragment.app.Fragment;
import android.view.LayoutInflater;
import android.view.View;
import android.view.ViewGroup;
import android.widget.TextView;
import six.ca.droiddailyproject.R;


/**
 * @copyright six.ca
 * Created by Xiaolin on 2016-11-28.
 */

public class FragmentOne extends Fragment {

    @Nullable
    @Override
    public View onCreateView(LayoutInflater inflater, ViewGroup container, Bundle savedInstanceState) {
        View view = inflater.inflate(R.layout.frag_one, container, false);
        TextView label = (TextView) view.findViewById(R.id.tv_label);
        label.setText(R.string.label_fragment_one);
        final Cart cart = new Cart(this);

        view.findViewById(R.id.btn_load).setOnClickListener(new View.OnClickListener() {
            @Override
            public void onClick(View v) {
//                cart.jumpDialogActivity();
                cart.jumpDialog();
//                Intent intent = new Intent(getActivity(), DialogActivity.class);
//                startActivityForResult(intent, 0);
            }
        });

        return view;
    }

    @Override
    public void onActivityResult(int requestCode, int resultCode, Intent data) {
        super.onActivityResult(requestCode, resultCode, data);
        System.out.println("xxl-frag");
        //do something...
    }

    @Override
    public void onDestroy() {
        System.out.println("xxl-FragmentOne-onDestroy");
        super.onDestroy();
    }
}

File: DroidDailyProject/app/src/main/java/six/ca/droiddailyproject/issues/scrollview/FragmentOne.kt
package six.ca.droiddailyproject.issues.scrollview

import android.os.Bundle
import com.google.android.material.tabs.TabLayout
import androidx.fragment.app.Fragment
import android.view.LayoutInflater
import android.view.View
import android.view.ViewGroup
import kotlinx.android.synthetic.main.fragment_issue_sv_one.*
import six.ca.droiddailyproject.R

/**
 * @CopyRight six.ca
 * Created by Heavens on 2019-01-12.
 */
class FragmentOne: androidx.fragment.app.Fragment() {

    override fun onCreateView(inflater: LayoutInflater, container: ViewGroup?, savedInstanceState: Bundle?): View? {

        return inflater.inflate(R.layout.fragment_issue_sv_one, container, false)
    }

    override fun onViewCreated(view: View, savedInstanceState: Bundle?) {
        super.onViewCreated(view, savedInstanceState)

        tabs.addTab(tabs.newTab().setText("OneOne"))
        tabs.addTab(tabs.newTab().setText("OneTwo"))
        tabs.addTab(tabs.newTab().setText("OneThree"))

        vp.adapter = fragmentManager?.let { PageAdapter(it, tabs.tabCount) }
        vp.addOnPageChangeListener(TabLayout.TabLayoutOnPageChangeListener(tabs))
        tabs.addOnTabSelectedListener(object : TabLayout.OnTabSelectedListener{
            override fun onTabReselected(p0: TabLayout.Tab) {
            }

            override fun onTabUnselected(p0: TabLayout.Tab) {
            }

            override fun onTabSelected(p0: TabLayout.Tab) {
                vp.currentItem = p0.position
            }

        })
    }
}